app/models/item_lookup.rb

```diff
@@ -8,12 +8,18 @@ def initialize(sku)
 
   def get_item(sku)
     tries = 0
-    while(current_price.nil? && tries < 20)
-      @item = client.lookup(sku).first
-      tries += 1
+    while current_price.nil?
```

Comment:
because we only need to retry if it fails.  If it succeeds then current_price != nil